BrentOzarULTD/SQL-Server-First-Responder-Kit
# Issue 1261

Fixes #1261 

Changes proposed in this pull request:
Adds missing index details to sp_BlitzCache and sp_BlitzQueryStore.

How to test this code:
Run a query with am missing index that has an impact >70%, you should see a new clickable column with the request.

Supports queries with multiple requests, and puts the highest impact queries first.

Has been tested on (remove any that don't apply):
 - Case-sensitive SQL Server instance
 - SQL Server 2016
